### ISSUE TITLE
Shorter logs missing in connect.comma,ai

### ISSUE THREAD

**Issue by mvl-boston** [commaai/connect#554]
### Describe the bug

A number of my shorter logs are missing in connect.comma.ai.   Usually they represent immediate car faults as I attempt to port new cars.

The lack of accessibility in connect.comma.ai is blocking my ability to upload rlogs to debug car port issues.

These logs are available in useradmin (but I can't request upload of rlogs from that site).

### Provide a route where the issue occurs

ad9840558640c31d/0000004d--daf90bf8ea

### openpilot version

0.9.9

**Comment by sshane** [comment]
Duplicate of https://github.com/commaai/connect/issues/60

**Comment by mvl-boston** [comment]
Hi all, a couple updates/questions:
1) Clicking on the connect link in useradmin doesn't work.  Connect opens up and says "no routes found in the selected timeframe"
2) is there a link to the "new connect" that isn't impacted by the lack of gps time?  Again, my primary goal is to tell the 3x to upload rlogs for these short/failed routes.

**Comment by incognitojam** [comment]
@mvl-boston Login at https://connect.aks.comma.ai, and then go to `https://connect.aks.comma.ai/dongleid/logid`, replacing `dongleid/logid` with the route ID
